src/data/bucket/symbol_bucket.js

```diff
@@ -61,32 +60,40 @@ const symbolInterfaces = {
         ]
     },
     collisionBox: { // used to render collision boxes for debugging purposes
-        layoutVertexArrayType: createVertexArrayType([
+        layoutAttributes: [
             {name: 'a_pos',     components: 2, type: 'Int16'},
             {name: 'a_extrude', components: 2, type: 'Int16'},
             {name: 'a_data',    components: 2, type: 'Uint8'}
-        ]),
+        ],
         elementArrayType: createElementArrayType(2)
     }
 };
 
-function addVertex(array, x, y, ox, oy, tx, ty, minzoom, maxzoom, labelminzoom, labelangle) {
+function addVertex(array, x, y, ox, oy, tx, ty, sizeData, minzoom, maxzoom, labelminzoom, labelangle) {
     array.emplaceBack(
-            // a_pos_offset
-            x,
-            y,
-            Math.round(ox * 64),
-            Math.round(oy * 64),
-
-            // a_texture_pos
-            tx / 4, // x coordinate of symbol on glyph atlas texture
-            ty / 4, // y coordinate of symbol on glyph atlas texture
-
-            // a_data
+        // a_pos_offset
+        x,
+        y,
+        Math.round(ox * 64),
+        Math.round(oy * 64),
+
+        // a_data
+        tx / 4, // x coordinate of symbol on glyph atlas texture
+        ty / 4, // y coordinate of symbol on glyph atlas texture
+        packUint8ToFloat(
             (labelminzoom || 0) * 10, // labelminzoom
-            labelangle, // labelangle
+            labelangle // labelangle
```

Comment:
Packing labelangle into 8 bits is really tight (less than 1 degree precision). Let's try to find some spare bits somewhere else.